src/libsyntax/feature_gate.rs

```diff
@@ -648,6 +645,8 @@ declare_features! (
     (accepted, repr_transparent, "1.28.0", Some(43036), None),
     // Defining procedural macros in `proc-macro` crates
     (accepted, proc_macro, "1.29.0", Some(38356), None),
+    // Allows use of the :vis macro fragment specifier
+    (accepted, macro_vis_matcher, "1.29.0", Some(41022), None),
```

Comment:
It should be `1.30.0`

Reply:
Oops sorry: I wondered about this after rebasing on top of master, but forge instructions said to look at forge for the next stable and that was 1.29.

Shall I send a PR to fix this up?

Reply:
Pinging reviewer: @cramertj

Reply:
Sure-- a PR to bump the version number would be appreciated :smile:

Reply:
Sorry for missing this the first time around.

Reply:
@cramertj sorry about that, here's the PR: https://github.com/rust-lang/rust/pull/53702